src/interfaces.d.ts

```diff
@@ -7,9 +7,22 @@ export interface Column<T> {
 
 export interface DataProperties<T> {
 	items: ItemProperties<T>[];
+	size: SizeDetails;
+	slice?: SliceDetails;
 	sort?: SortDetails[];
 }
 
+export interface HasBufferRows {
+	/**
+	 * @default 10
```

Comment:
This seems like not the right place to document default values - is it possible to document them in the right place and get the right generated doc output?